railties/lib/rails/commands/task_builder/test.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'common'
+require 'rake'
+
+module Rails
+  module Commands
+    module TaskBuilder
+      # This is a wrapper around all Rails test tasks, including:
+      #   rails test
+      #   rails test:db
+      class Test
```

Comment:
Good suggestion. Let me know if you have any questions about the overall structure.